.github/workflows/lint.yml

```diff
@@ -1,7 +1,7 @@
 on:
   pull_request:
     branches:
-      - v1
+      - main
```

Comment:
Looks like we might need to update some owlbot config to avoid getting this overwritten. CC @shobsi